docs/agents/chopper_mc2000b.rst

```diff
@@ -0,0 +1,56 @@
+.. highlight:: rst
+
+.. _chopper_mc2000b_agent:
+
+========================
+Chopper MC2000B Agent
+========================
+
+The MC2000B Chopper Agent is an OCS agent which helps monitor input and output
+frequencies of the chopper, and sends commands to set the frequency of the chopper,
+as well as other features such as the bladetype and reference modes of the device.
+
+Dependencies
+------------
+
+Python Packages:
```

Comment:
I'd say remove the ":" and make this a subsection header with an underline of backticks.